components/input-textUt/text-ut.css

```diff
@@ -0,0 +1,16 @@
+.grey-text {
+    color: grey;
```

Comment:
use color variables here.

Reply:
I have changed it, thanks for the review